src/controllers/userController.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express");
+
+let users = [];
+
+// Get all Users
+exports.getAllUsers = (req, res) => {
+  res.status(200).json(users);
+};
+
+// Get user by ID
+exports.getUserById = (req, res) => {
+  const userId = req.params.userId;
+  const user = users.find((user) => user.id === parseInt(userId));
+  if (user) {
+    res.status(201).json(user);
+  } else {
+    res.status(404).json({ message: `Not found user with id ${userId}` });
+  }
+};
+
+// Create new user
+exports.createUser = (req, res) => {
+  const newUser = {
+    id: users.length + 1,
+    name: req.body.name,
```

Comment:
It's better to use destructurezation for the req.body

Reply:
Done